json_spec/README.md

```diff
@@ -1,16 +1,2589 @@
+    </tr>
+</table>
+
+
+#### Test Step Artifacts
+
+A [_testStepArtifact_](#test-step-artifacts) refers to data and information associated with a particular sub-sequence of a [**diagnostic's**](#def-diagnostic) overall execution.
+
+<table>
+    <tr>
+        <td><strong>Attribute</strong></td>
+        <td><strong>Type</strong></td>
+        <td><strong>Required</strong></td>
+        <td><strong>Description</strong></td>
+    </tr>
+    <tr>
+        <td><em>testStepId</em></td>
+        <td>string</td>
+        <td><strong>Yes</strong></td>
+        <td>A <a href="#def-uniqueidentifier"><strong>unique identifier</strong></a> for the <a href="#def-teststep"><strong>test step</strong></a>. This identifier is used to associate any additional messages with the <a href="#def-teststep"><strong>test step</strong></a>.</td>
+    </tr>
+    <tr>
+        <td><strong><em>&lt;artifact&gt;</em></strong></td>
+        <td>JSON: object<br/>Message: various</td>
+        <td><strong>Yes</strong></td>
+        <td>The artifact test step message.</td>
+    </tr>
+</table>
+
+The following artifacts are supported attributes of a test step artifact:
+
+<table>
+    <tr>
+        <td><strong>Object</strong></td>
+        <td><strong>Description</strong></td>
+    </tr>
+    <tr>
+        <td><a href="#teststepstart"><em>TestStepStart</em></a></td>
+        <td>Information pertaining to the starting condition of a <a href="#def-teststep"><strong>test step</strong></a>.</td>
+    </tr>
+    <tr>
+        <td><a href="#teststepend"><em>TestStepEnd</em></a></td>
+        <td>Information pertaining to the ending condition of a <a href="#def-teststep"><strong>test step</strong></a>.</td>
+    </tr>
+    <tr>
+        <td><a href="#measurement"><em>Measurement</em></a></td>
+        <td>Information concerning a discrete <strong>measurement</strong> or telemetry from a <a href="#def-dut"><strong>device under test</strong></a>.</td>
+    </tr>
+    <tr>
+        <td><a href="#measurementseriesstart"><em>MeasurementSeriesStart</em></a></td>
+        <td>Information concerning the beginning of a <strong>measurement series</strong>.</td>
+    </tr>
+    <tr>
+        <td><a href="#measurementseriesend"><em>MeasurementSeriesEnd</em></a></td>
+        <td>Information concerning the ending of a <strong>measurement series</strong>.</td>
+    </tr>
+    <tr>
+        <td><a href="#measurementserieselement"><em>MeasurementSeriesElement</em></a></td>
+        <td>Information concerning a discrete observation contained in a <strong>measurement</strong> or <strong>measurement series</strong>.</td>
+    </tr>
+    <tr>
+        <td><a href="#diagnosis"><em>Diagnosis</em></a></td>
+        <td>The identification of the status of a <a href="#def-dut"><strong>device or system under test</strong></a> based on the examination of measurements and <a href="#def-symptom"><strong>symptoms</strong></a>.</td>
```

Comment:
same comment as above saying device or system under test makes it sound like we are making a distinction and we haven't defined the difference between a system vs. device under test.  Stick with device?

Reply:
i left "system under test" just in 2 cases, where it's definitions. everywhere else it's "device under test"

Reply:
done. please check/resolve